examples/52_tanstack-router/src/app.tsx

```diff
@@ -0,0 +1,27 @@
+import { Router } from './ts-router';
```

Comment:
Just "router"?. "ts" is confusing with TypeScript.
```suggestion
import { Router } from './router';
```